packages/matter.js/src/net/UdpMulticastServer.ts

```diff
@@ -73,14 +75,15 @@ export class UdpMulticastServer {
                 await Promise.all(
                     ips.map(async ip => {
                         const iPv4 = isIPv4(ip);
+                        const broadcastTarget = iPv4 ? this.broadcastAddressIpv4 : this.broadcastAddressIpv6;
+                        if (broadcastTarget == undefined) {
+                            // IPv4 but disabled, so just resolve
+                            return Promise.resolve();
```

Comment:
I think you can just return here, Promise.all() takes any thenable object so undefined should be OK

Reply:
It is in fact even easier ... it is an "asny function" ... so it will always return a promise :-)